products/epinio/models/services.js

```diff
@@ -8,16 +8,16 @@ import EpinioResource from './epinio-resource';
 export default class EpinioService extends EpinioResource {
   get links() {
     return {
-      update: this.getUrl(),
-      self:   this.getUrl(),
-      remove: this.getUrl(),
-      create: this.getUrl(this.meta?.namespace, null), // ensure name is null
+      update: this.getUrl(this.meta?.namespace, this.meta?.name),
```

Comment:
update, self and remove won't need these params as they're the functions defaults